starting_package/openvino_workbench/container.py

```diff
@@ -17,153 +17,228 @@
-    elif is_container_running(docker_client, container_name):
-        print(f'A container with the name "{container_name}" is running - there is no need to restart it.')
-        print('Aborting.')
-        sys.exit(1)
-
-    # Get and restart the container
-    container = docker_client.containers.get(container_name)
-    container.start()
-
-    # Wait for it to be ready
-    for pre_message, complete_message in zip(PRE_STAGE_MESSAGES, STAGE_COMPLETE_MESSAGES):
-        wait_for_stage_to_complete(pre_message, re.compile(complete_message), docker_client, container_name)
-
-    print_finishing_message(docker_client, container_name, is_detached)
-
-    if is_detached:
-        sys.exit(0)
-
-    attach_to_container_and_display_logs(docker_client, container_name)
-
-
-def attach_to_container_and_display_logs(docker_client: DockerClient, container_name: str):
-    for log in docker_client.api.attach(container=container_name, stream=True):
-        print(log.decode('utf-8'), sep='', end='')
+    WORKBENCH_READY_MESSAGE, LOG_FILE, EXAMPLE_COMMAND, INTERNAL_PORT, ABORTING_EXIT_MESSAGE
+
+
+class Container:
+    def __init__(self, docker_client: DockerClient, logger: logging.Logger, config: dict):
+        self._client = docker_client
+        self._logger = logger
+        self.config = config
+        self.container_name = self.config['name']
+        self._is_present = self._is_container_present()
+        self._is_running = self._is_container_running()
+
+    def start(self, detached: bool, network_name: str, network_alias: str):
+        self._logger.info('Starting container.')
+
+        if self._is_present:
+            if self._is_running:
+                self._logger.info('Container with the specified name is already running.')
+                public_port = self._get_public_port()
+                print(
+                    f'A container with the name "{self.container_name}" is running.'
+                    f'\n\nOpen the browser and navigate to the http://127.0.0.1:{public_port}\n'
+                    f'\nPath to the log file: {LOG_FILE}')
+                sys.exit(1)
+
+            print(f'ERROR: A container with the name "{self.container_name}" is present on the machine and '
+                  'is not running.'
+                  '\nYou can restart the container using the following command:'
+                  f'\n\n\topenvino-workbench --restart {self.container_name}')
+            new_name = self._generate_container_name()
+            new_port = self._generate_container_port()
+            if new_name:
+                message = (f'\n\nOr start a new one using the following command:'
+                           f'\n\n\topenvino-workbench --container-name {new_name}')
+                if new_port:
+                    message += f' --port {new_port}'
+                print(f'{message}\n')
+            else:
+                print('Example command: '
+                      '\n\n\topenvino-workbench --container-name NEW_NAME '
```

Comment:
I would suggest moving `openvino-workbench` string somewhere to class-level or even global constants for further reuse in such messages. Possibly CLI parameter name (like `--container-name`) can be also save to constants - let's use rule of three (if something is repeated 3 or more times let's move it to variable/constant and reuse)